src/stories/packages/components/InfoBox.stories.ts

```diff
@@ -0,0 +1,53 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import { InfoBox } from '@terra-ui-packages/components';
+import { allIconIds } from '@terra-ui-packages/components';
+
+const meta: Meta<typeof InfoBox> = {
+  title: 'Packages/Components/InfoBox',
+  component: InfoBox,
+  parameters: {
+    layout: 'centered',
+  },
+  tags: ['autodocs'],
+  argTypes: {
+    children: {
+      control: 'text',
+      description: 'Defines the popup content to appear when the InfoBox is clicked.',
+    },
+    icon: {
+      control: 'select',
+      options: allIconIds,
+      description: 'The ID of an icon to be used from IconLibrary',
+    },
+    side: {
+      control: 'select',
+      options: ['top', 'bottom', 'right', 'left'],
```

Comment:
it'd be better if these could come directly from the `Side` type, but I couldn't figure out how to do that -- has anyone done that before?

Reply:
I agree that would be nice. I don't know if we have figured out how to iterate through types. @nawatts any suggestions?

Reply:
There's not really a way to iterate over values of a union type, since the types are removed at compile time.

It's easy to go the other way, and construct a union type out of an array of values.
```ts
const values = ['foo', 'bar', 'baz'] as const;
type Value = typeof values[number];
```

You can use TypeScript to verify that an array contains all values of a union type.
https://stackoverflow.com/questions/44480644/string-union-to-string-array